addons/pos_restaurant/static/src/app/screens/product_screen/product_screen.xml

```diff
@@ -14,10 +14,16 @@
                         <span class="d-block">Order</span>
                         <small><t t-esc="nbrOfChanges"/> changes</small>
                     </button>
+                    <button
+                            class="h-100 button btn btn-light btn-lg d-flex flex-row align-items-center justify-content-center"
```

Comment:
![image](https://github.com/user-attachments/assets/378456b7-1c2a-4380-8986-be8764392d26)
This does not feel like a button, what do you think

Reply:
i will change that